### PR TITLE
Load Local

### PR DESCRIPTION
**Pull request by imeoer**

### CODE REVIEW

File: vendor/src/github.com/docker/engine-api/client/image_load_local.go [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+	"path/filepath"
+	"strconv"
+
+	"github.com/docker/engine-api/types"
+	"github.com/hyperhq/hypercli/image"
+	"github.com/hyperhq/hypercli/pkg/archive"
+	"github.com/hyperhq/hypercli/pkg/progress"
+	"github.com/hyperhq/hypercli/pkg/streamformatter"
+	"github.com/hyperhq/hypercli/pkg/symlink"
+
```
Comment (carmark) [inline vendor/src/github.com/docker/engine-api/client/image_load_local.go]: no needed space.

File: vendor/src/github.com/docker/engine-api/client/image_load_local.go [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+	var diffRet struct {
+		ExistLayers []string `json:"existLayers"`
+	}
+	if err := json.Unmarshal(body, &diffRet); err != nil {
+		return nil, nil, err
+	}
+	return diffRet.ExistLayers, layerPaths, nil
+}
+
+func (cli *Client) ImageLoadFromTar(ctx context.Context, out io.Writer, tr io.Reader, quiet bool) (*types.ImageLoadResponse, error) {
```
Comment (carmark) [inline vendor/src/github.com/docker/engine-api/client/image_load_local.go]: cli has `Out` field.

File: vendor/src/github.com/docker/engine-api/client/image_load_local.go [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+	}()
+
+	return &types.ImageLoadResponse{
+		Body: readCloser{io.MultiReader(pr, resp.Conn), resp.Conn},
+		JSON: true,
+	}, nil
+}
+
+// ImageDiff diff an image layers with local and imaged
+func (cli *Client) ImageLoadFromDaemon(ctx context.Context, out io.Writer, name string, quiet bool) (*types.ImageLoadResponse, error) {
```
Comment (carmark) [inline vendor/src/github.com/docker/engine-api/client/image_load_local.go]: same as above.

File: vendor/src/github.com/docker/engine-api/client/image_load_local.go (hunk 2 above)
Comment (imeoer) [inline vendor/src/github.com/docker/engine-api/client/image_load_local.go]: @carmark Seems not found.

**Comment by carmark** [comment]
LGTM, but could you please combine the commits?

**Pull request merged**